### PR TITLE
Various documentation corrections

### PR DESCRIPTION
**Pull request by ChrisMM**
Of the list of items I sent in email, these changes are only the Corrections and Issues identified, **not** the Suggestions, Questions, nor Conflicts.

Hopefully, the number of files being changed is not too many because each commit only has one change for easy review: each commit to this branch has a single bullet point explaining the change and why.

Any questions, please let me know. Thanks

@kevgo @mcharawi

### CODE REVIEW

**Comment by mcharawi** [comment]
I stepped through the changed links, everything looks good.

File: tutorial/part_2/part_2a/01_exo_tool.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ We will look at it in a minute.
 First, the takeaway of this chapter:
 
 > The Exosphere framework provides a powerful command-line application
-> that allows to work on and operate Exosphere applications.
+> that allows you to work on and operate Exosphere applications.
```
Comment (kevgo) [inline tutorial/part_2/part_2a/01_exo_tool.md]: Rob pointed out the same issue... I could swear this read okay without it, but fine..  :)

**Comment by kevgo** [comment]
Great catches in there, thanks for the vigilance and going through all this!

File: tutorial/part_2/part_2a/01_exo_tool.md (hunk 1 above)
Comment (ChrisMM) [inline tutorial/part_2/part_2a/01_exo_tool.md]: 👀

**Pull request closed without merging**